bubble-wrap.yaml

```diff
@@ -787,16 +787,16 @@ scene:
         color: *scene1
 
 layers:
-    # Map overlays for styling the server response (using special source layer names) for route line, find_me location gem, and search result pins
-    route:
-        data: { source: route }
+    # Map overlays for styling the server response (using special source layer names) for mz_route_line, mz_current_location location gem, and mz_search_result result pins
```

Comment:
These should be plain english versions of the features, not variable names. 

Update to:

* `for route line, current location, and search result pins`